### PR TITLE
Split side planks

### PR DESCRIPTION
**Pull request by annawoodard**
This PR splits side planks into separate exercises so that users can track each side independently.

I was unable to build without 1f5108a and 8fe82b3-- let me know if you prefer me to split those out so you can consider them separately.

### CODE REVIEW

**Comment by mazurio** [comment]
Hi @annawoodard,

The first two commits are good. I think those tests are very out of date - they were added a while ago before UI changes. It might be worth changing the video of the second side plank - I was thinking about mirroring the mp4 - what do you think?

**Comment by annawoodard** [comment]
Great point about the video @mazurio, sorry I missed that! Should be taken care of in 202e9ea:

screen shot 2017-01-06 at 10 04 22 pm

screen shot 2017-01-06 at 10 07 36 pm

File: app/src/main/res/raw/bodyweight_fitness_recommended_routine.json [hunk 1]
```diff
@@ -115,12 +115,21 @@
       "defaultSet": "timed"
     },
     {
-      "exerciseId": "exercise8",
-      "title": "Side Plank (Both Sides)",
+      "exerciseId": "exercise8left",
```
Comment (mazurio) [inline app/src/main/res/raw/bodyweight_fitness_recommended_routine.json]: Can we keep this id as `exercise8`?

File: app/src/main/res/raw/bodyweight_fitness_recommended_routine.json [hunk 2]
```diff
@@ -115,12 +115,21 @@
...
+      "title": "Left Side Plank",
       "description": "1x(10-60s) hold",
       "type": "exercise",
       "youTubeId": "xIFWxQgFMiE",
-      "videoId": "bodyweight_fitness_side_plank",
+      "videoId": "bodyweight_fitness_left_side_plank",
+      "defaultSet": "timed"
+    },
+    {
+      "exerciseId": "exercise8right",
```
Comment (mazurio) [inline app/src/main/res/raw/bodyweight_fitness_recommended_routine.json]: And this one should be `f4f27acd-fd69-40f8-a459-a9f02c56ed03`.

**Review by mazurio — COMMENTED** [review]
That looks good - the video looks really good now! I only added tiny comments about id's. UUID is now used everywhere for new exercises (see Molding Mobility or SS). For the first side plank we would like to leave the same id as it will allow users to see historical data with graphs otherwise all this data for side planks will be lost. Thanks!

**Comment by annawoodard** [comment]
@mazurio Ah, thanks for pointing that out! Should be fixed in 36cb9ef.

**Review by mazurio — APPROVED** [review]

**Comment by mazurio** [comment]
Thanks @annawoodard

**Pull request merged**